docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2498 @@
+[[breaking-141757]]
+.Updates bulk action API to return actionId instead of agents success for consistency
+[%collapsible]
+====
+*Details* +
+To make bulk action responses consistent, returns `actionId` instead of agent ids with `success: True` or `success: False` results. For more information, refer to {kibana-pull}141757[#141757].
+
+*Impact* +
+When you use `FleetBulkResponse`, you now receive only `actionId` responses. 
+====
+      
+[discrete]
+[[breaking-139431]]
+.Removes filter validation for ad-hoc data views
+[%collapsible]
+====
+*Details* +
+Filters associated with unknown data views, such as deleted data views, are no longer automatically disabled. For more information, refer to {kibana-pull}139431[#139431].
+
+*Impact* +
+Filters associated with unknown data views now display a warning message instead of being automatically disabled.
+====
+      
+[discrete]
+[[breaking-138677]]
+.Removes the `package_policies` field from the agent policy saved object
+[%collapsible]
+====
+*Details* +
+The bi directionnal foreign key between agent policy and package policy has been removed. For more information, refer to {kibana-pull}138677[#138677].
+
+*Impact* +
+The agent policy saved object no longer includes the `package_policies` field.
+====
+// end::notable-breaking-changes[]
+      
+[float]
+[[features-8.5.0]]
+=== Features
+{kib} 8.5.0 adds the following new and notable features.
+
+Alerting::
+* 137988 browser fields UI {kibana-pull}140516[#140516]
+* Show alerts count {kibana-pull}140473[#140473]
+* Adds the ability to allows users to assign other users to cases {kibana-pull}140208[#140208]
+* Ability run a rule on-demand {kibana-pull}139848[#139848]
+* Ability to bulk update API keys for alerting rules {kibana-pull}139036[#139036]
+* Index threshold alert can't use unsigned long data type {kibana-pull}138452[#138452]
+* Category fields endpoint {kibana-pull}138245[#138245]
+* Index threshold alert UI does not fill index picker with data streams {kibana-pull}137584[#137584]
+
+APM::
+* Support Input Packages {kibana-pull}140035[#140035]
+* Display kubernetes metadata in service icons popup and instance accordion {kibana-pull}139612[#139612]
+* AWS lambda metrics api {kibana-pull}139041[#139041]
+
+Discover::
+* Adds support for storing time with saved searches {kibana-pull}138377[#138377]
+* Enables tags for saved searches {kibana-pull}136162[#136162]
+
+Elastic Security::
+For the Elastic Security 8.5.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+* Agent activity {kibana-pull}140510[#140510]
+* Config options added to policy {kibana-pull}139421[#139421]
+* Moving action batching to async {kibana-pull}138870[#138870]
+* Tagging of integration assets {kibana-pull}137184[#137184]
+
+Infrastructure::
+Inital hosts page {kibana-pull}138173[#138173]
+
+Lens & Visualizations::
+Adds the ability to duplicate layers in *Lens* {kibana-pull}140603[#140603]
```

Comment:
@ninoslavmiskovic @ghudgins should we mention query based annotations and/or ad-hoc data views here?